.golangci.yml

```diff
@@ -33,7 +33,7 @@ linters-settings:
   nakedret:
     max-func-lines: 0
   revive:
-    min-confidence: 0.8
+    confidence: 0.8
```

Comment:
@mmorel-35 they introduced a JSONSchema validation that is detecting this value as incorrect. It was introduced in #2947

I _think_ the golangci lint is detecting this as an error since https://github.com/golangci/golangci-lint/pull/5448